style.css

```diff
@@ -82,6 +82,19 @@ img {
   column-gap: 30px;
 }
 
+.mobile-image {
+  display: none;
+}
+.desktop-image {
+  width: 100%;
+}
+.image-box {
+  max-width: 780px;
+  max-height: 324px;
+  overflow: hidden;
+  aspect-ratio: 780/324;
```

Comment:
won't this stretch the image?
Can't we use `object-fit` instead of this ?

Reply:
I have tried a small image and it is not stretching it.
there is no need to use object fit as it is maintaining aspect ratio of image